secure_message/resources/messages.py

```diff
@@ -138,26 +136,6 @@ def _get_user_name(user, message):
         return user_name
 
 
-class MessageById(Resource):
-    """Get and update message by id"""
-
-    @staticmethod
-    def get(message_id):
-        """Get message by id"""
-        logger.info("Getting individual message", user_uuid=g.user.user_uuid)
-        # check user is authorised to view message
-        message_service = Retriever()
-        message = message_service.retrieve_message(message_id, g.user)
-
-        message = add_users_and_business_details([message])[0]
-        if Authorizer().can_user_view_message(g.user, message):
-            return jsonify(message)
-        result = jsonify({'status': 'error'})
-        result.status_code = 403
-        logger.error('Error getting message by ID', msg_id=message_id, status_code=result.status_code)
-        return result
-
-
 class MessageModifyById(Resource):
```

Comment:
If your removing the use of `MessageById` there would be no need for this either? or is that for the next iteration of this?

Reply:
We still need message modify by id because we need to change the labels (read -> unread for example)